packages/quantification/src/__tests__/quantification.test.ts

```diff
@@ -248,12 +249,15 @@ describe('getUnadjustedGrandfatheredTonnesPerYear', () => {
 
 describe('getQuantificationSummary', () => {
   it('will get the tonnes that are grandfatherable given a COMET output file for 5 grandfatherable years', () => {
-    expect(
-      getQuantificationSummary({
-        data: GRANDFATHERABLE_YEARS_OUTPUT,
-        maxNumberOfGrandfatheredYears: 5,
-      })
-    ).toStrictEqual<ReturnType<typeof getQuantificationSummary>>({
+    const { parsedJsonOutput } = parseYearlyMapUnitData({
+      rawJsonOutput: GRANDFATHERABLE_YEARS_OUTPUT,
+    });
+    const result = getQuantificationSummary({
+      parsedJsonOutput,
+      maxNumberOfGrandfatheredYears: 5,
+    });
```

Comment:
This approach for testing is workable, but not ideal since tests rely on `parseYearlyMapUnitData(..)` and are not atomic. 

I plan to update these tests so that calls to `parseYearlyMapUnitData(..)` are not needed, but would prefer to discuss the spirit of the PR and its changes before investing that effort. 

In the meantime, I'll open this PR as a draft to have the discussion about the general approach

Reply:
I addressed this with a new update